runtime/themes/meliora.toml

```diff
@@ -69,6 +69,7 @@
 "markup.list" = { fg = "blue" }
 "markup.bold" = { fg = "magenta" }
 "markup.italic" = { fg = "blue" }
+"markup.strikethrough" = { modifiers = ["crossed_out"] }
```

Comment:
@ramojus I noticed that your theme doesn't specify the `bold` or `italic` modifiers for markup. Is this on purpose?

I can add them if you want, or remove the `crossed_out` modifier again.

Reply:
Yes you can add them, thank you!